plugin/manager_linux.go

```diff
@@ -128,3 +128,21 @@ func (pm *Manager) disable(p *plugin) error {
 	pm.save()
 	return nil
 }
+
+// Shutdown stops all plugins and called during daemon shutdown.
+func (pm *Manager) Shutdown() error {
```

Comment:
We return the error now. But its purely for logging purposes. The daemon shutdown should not fail, if any of the sub components fail to stop. It should continue its best-effort shutdown, as it does today.